src/main/java/uk/gov/justice/hmpps/prison/api/model/Alert.java

```diff
@@ -81,6 +82,10 @@ public class Alert {
     @JsonProperty("dateExpires")
     private LocalDate dateExpires;
 
+    @Schema(description = "Datetime the alert was last modified", example = "2020-08-20T12:34:56")
```

Comment:
not sure if this should be `RequiredMode.NOT_REQUIRED` too - can the field be `null`?